server/apollo.go

```diff
@@ -191,7 +193,10 @@ func (apollo *Apollo) GetPath(key string) string {
 //		return fmt.Errorf("cannot parse body: %w", err)
 //	}
 func (apollo *Apollo) ParseBody(v interface{}) error {
-	return render.DecodeForm(apollo.Request.Body, v)
+	if apollo.decoder == nil {
+		apollo.decoder = schema.NewDecoder()
+	}
+	return apollo.decoder.Decode(v, apollo.Request.PostForm)
```

Comment:
Je moet blijkbaar eerst `apollo.Request.ParseForm()` oproepen, anders is `PostForm` leeg.

Reply:
En met deze library moet de naam van de input hetzelfde zijn als de naam van het veld in de struct (ik zal daarvoor een PR aanmaken in wachtplanner, daar is dat bv niet het geval bij het aanmaken van een hospital)

Reply:
`parseForm` moet je doen om de data in PostForm in te laden, dat is bij chi/render ook zo (is gewoon hoe requests werken in Go) 😛 ik dacht dat we dat al sowieso deden maar blijkbaar doen we dit momenteel enkel als je het CSRF token checkt en anders niet, dus ik zal het hier ook toevoegen zodat de library dat altijd automatisch doet als je dat nodig hebt 😛

Reply:
Je kan velden hier toch ook hernoemen? Ze gebruiken gewoon een "schema" tag ipv "form". Je kan ook extra dingen doen zoals `schema:"name,required"` om te zeggen dat een veld verplicht is of `schema:"-"` om te zeggen dat die data niet uit de post mag komen